src/vs/vscode.d.ts

```diff
@@ -4325,6 +4330,29 @@ declare module 'vscode' {
 		resolveTask(task: Task, token?: CancellationToken): ProviderResult<Task>;
 	}
 
+	export enum LogLevel {
```

Comment:
everywhere! https://github.com/Microsoft/vscode/wiki/Extension-API-guidelines#jsdoc

Reply:
I know this matches the enum we defined in VS Code, but I think we should make `Off = 0`. That way, if a js user writes:

```ts
if (logger.level) {
     ...
}
```

We do the more correct thing

Reply:
I dunno, the user doesn't need to do that, and that will put the levels out of order. And translating between the enums will be weird.